ini.go

```diff
@@ -111,8 +111,15 @@ type LoadOptions struct {
 	KeyValueDelimiters string
 	// PreserveSurroundedQuote indicates whether to preserve surrounded quote (single and double quotes).
 	PreserveSurroundedQuote bool
+	// Debug indicates whether to collect debug information (currently only useful to debug parsing Python-style multiline values).
+	Debug DebugFunc
```

Comment:
I think this field should also be called `DebugFunc`, because it is a function now.

Reply:
makes sense